### PR TITLE
Add compatibility checks (min version)

### LINKED ISSUE #142: Disable install button when add-on is not compatible with the Firefox version (min/max)
Once https://github.com/mozilla/addons-blog/issues/140 lands, we'll have to check the min/max version constraints when the browser is Firefox because some add-ons might not be compatible with it.

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #142

### CODE REVIEW

**Comment by willdurand** [comment]
(updated addons-moz-compare, it's ready now)

File: src/assets/js/dynamic-addon-cards.js [hunk 1]
```diff
@@ -130,9 +133,29 @@
             }
           }
 
-          // TODO: we will need to check the min version since the max version
-          // is usually not a problem (AMO does not disable the install button
-          // when we are over the max version).
+          if (!isIncompatible) {
+            const compatibility = current_version.compatibility[clientApp];
```
Comment (muffinresearch) [inline src/assets/js/dynamic-addon-cards.js:137]: Presumably this data is generally guaranteed and if those expectations aren't met the catch would cover it?

**Review by muffinresearch — APPROVED** [review]
r+wc

File: src/assets/js/dynamic-addon-cards.js (hunk 1 above)
Comment (willdurand) [reply src/assets/js/dynamic-addon-cards.js:137]: `current_version` is checked above but `.compatibility` isn't necessarily checked all the time so I'll add a test case and a condition here.

**Pull request merged**